usaspending_api/search/delta_models/award_search.py

```diff
@@ -0,0 +1,96 @@
+award_search_sql_string = r"""
+CREATE OR REPLACE TABLE {DESTINATION_TABLE} (
+    treasury_account_identifiers ARRAY<INTEGER>,
+    award_id LONG NOT NULL,
+    category STRING,
+    type STRING,
+    type_description STRING,
+    generated_unique_award_id STRING,
+    display_award_id STRING,
+    update_date TIMESTAMP,
+    piid STRING,
+    fain STRING,
+    uri STRING,
+    award_amount NUMERIC(23, 2),
+    total_obligation NUMERIC(23, 2),
+    description STRING,
+    total_obl_bin STRING,
+    total_subsidy_cost NUMERIC(23, 2),
+    total_loan_value NUMERIC(23, 2),
+    recipient_hash BINARY,
+    recipient_levels ARRAY<STRING>,
+    recipient_name STRING,
+    recipient_unique_id STRING,
+    parent_recipient_unique_id STRING,
+    recipient_uei STRING,
+    parent_uei STRING,
+    business_categories ARRAY<STRING>,
+    action_date STRING,
```

Comment:
`action_date` as well as the five dates below should be `DATE` fields, not `STRING`